provider/import.go

```diff
@@ -176,3 +185,13 @@ func Import(ctx context.Context, p Provider, hcl, tfstate writer.Writer, f *filt
 
 	return nil
 }
+
+func getKey(s *terraform.InstanceState) string {
+	// google special case
+	// sometimes, the resource is referred by its self_link (https://...)
+	// so we need to identify if we reference the ressource by its ID or its selfLink (the key)
+	if _, ok := s.Attributes["self_link"]; ok {
+		return "self_link"
+	}
+	return "id"
```

Comment:
I've commented it to @tormath1 on PM that this could be much more things than just the `id`, on an `aws_instance` the relations could be done with https://www.terraform.io/docs/providers/aws/r/instance.html#attributes-reference any of those. But maybe for now only the ID it's the important one.

Reply:
Among the list we want for sure - and that we already widely use (some services I can think of):
`id` (general reference), `arn` (iam,alb,etc), `name` (iam,r53),  `fqdn` (r53)

Reply:
Well, my current thought would be to have an `AttributesReference` associated to a `Resource`. This attribute would be simply feed by https://github.com/cycloidio/tfdocs.

Interpolation would now be made there: 
https://github.com/cycloidio/terracognita/blob/5709caa05628bc41ad1d1a49b462da93bfa47a5d/provider/import.go#L92 since we have all the `resources`.

Then looping on the resources, if a value of resources.AttributesReferences[*] match with a value in the `resources`, we interpolate.
WDYT ?

Reply:
Well, my current thought would be to have an `AttributesReference` associated to a `Resource`. This attribute would be simply feed by https://github.com/cycloidio/tfdocs.

Interpolation would now be made there: 
https://github.com/cycloidio/terracognita/blob/5709caa05628bc41ad1d1a49b462da93bfa47a5d/provider/import.go#L92 since we have all the `resources`.

Then looping on the resources, if a value of resources.AttributesReferences[*] match with a value in the `resources`, we interpolate.
WDYT ?